src/scripts/options/csv.ts

```diff
@@ -0,0 +1,41 @@
+import fs from "fs/promises";
```

Comment:
Let's be more precise with the name. As we use a free function, the only thing exported: generateOptionReport.ts. If using a class: OptionReport.ts (which would have a single public method: optionReport.generate())